docs/components/modeler/bpmn/inclusive-gateways/inclusive-gateways.md

```diff
@@ -0,0 +1,90 @@
+---
+id: inclusive-gateways
+title: "Inclusive gateway"
+description: "An inclusive gateway (or OR-gateway) allows you to make a decision based on data."
+---
+
+:::note
+
+Currently, it's only add support for the diverging (i.e. splitting, forking) inclusive gateway, it's not add support for the converging (i.e. merging, joining) inclusive gateway.
+
+- The diverging inclusive gateway adds a lot of value, even without the converging behavior.
+- In addition, a combination of parallel and exclusive gateways can be used to merge the flows again.
+
+:::
+
+An inclusive gateway (or OR-gateway) allows you to make a decision based on data (i.e. on process instance variables).
+
+![process](assets/inclusive-gateway.png)
```

Comment:
❌ All images must have descriptive alt-texts according to the [styleguide](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/technical-writing-styleguide.md#formatting-organization-and-structure-for-implementation-steps). Please replace `process` with a full description of what the image shows. This improves the accessibility of the docs.

❌ These images were useful for the PR, but don't provide the same level of information as the images on other gateway pages (i.e. abstract task names, no labels on the gateway, and sequence flows. It would be best to follow our own best practices on [labeling elements](https://docs.camunda.io/docs/next/components/best-practices/modeling/creating-readable-process-models/#labeling-bpmn-elements). Please replace the images, so they:
 - are labeled according to our [best practices](https://docs.camunda.io/docs/next/components/best-practices/modeling/creating-readable-process-models/#labeling-bpmn-elements)
 - are sized according to our [styleguide](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/technical-writing-styleguide.md#formatting-organization-and-structure-for-implementation-steps) (i.e. All images must be appropriately sized). I'm unsure what the exact limitations are, but the images are very large compared to the rest of the text right now. When resizing the images, please take into account